blog/2023-06-08-Advisory-committee-member

```diff
@@ -0,0 +1,37 @@
+---
+author: Advisory Committee
+title: "Join OpenRefine's Advisory Committee"
+slug: 2023/06/08/Advisory-committee-member
```

Comment:
```suggestion
```

We don't need to put slugs, they are generated automatically. The reason why there are some for older posts is to preserve the URLs from before the Docusaurus migration, but for new ones it's not needed.